src/clib/old_tests/enkf/test_gen_kw_logarithmic.cpp

```diff
@@ -1,120 +0,0 @@
-
-namespace enkf_main {
-void ecl_write(const ensemble_config_type *ens_config,
-               const char *export_base_name, const char *run_path, int iens,
-               enkf_fs_type *fs);
-
-} // namespace enkf_main
-
-void verify_parameters_txt() {
-    int buffer_size = 0;
-    char *file_content = util_fread_alloc_file_content(
-        "simulations/run0/parameters.txt", &buffer_size);
-
-    stringlist_type *token_list =
-        stringlist_alloc_from_split(file_content, " \n");
-    double value = stringlist_iget_as_double(token_list, 5, NULL);
-
-    test_assert_true(value > 0.0);                            //Verify precision
-    test_assert_true(NULL != strstr(file_content, "LOG10_")); //Verify log entry
-
-    stringlist_free(token_list);
-    free(file_content);
-}
-
-void test_write_gen_kw_export_file(res_config_type *res_config) {
-    enkf_fs_type *init_fs =
-        enkf_fs_create_fs("new_fs", BLOCK_FS_DRIVER_ID, true);
-    ensemble_config_type *ens_config =
-        res_config_get_ensemble_config(res_config);
-    enkf_node_type *enkf_node =
-        enkf_node_alloc(ensemble_config_get_node(ens_config, "MULTFLT"));
-    enkf_node_type *enkf_node2 =
-        enkf_node_alloc(ensemble_config_get_node(ens_config, "MULTFLT2"));
-    test_assert_true(enkf_node_get_impl_type(enkf_node) == GEN_KW);
-    test_assert_true(enkf_node_get_impl_type(enkf_node2) == GEN_KW);
-
-    {
-        gen_kw_type *gen_kw = (gen_kw_type *)enkf_node_value_ptr(enkf_node);
-        gen_kw_type *gen_kw2 = (gen_kw_type *)enkf_node_value_ptr(enkf_node2);
-
-        {
-            rng_type *rng = rng_alloc(MZRAN, INIT_DEFAULT);
-            const enkf_config_node_type *config =
-                enkf_node_get_config(enkf_node);
-            const int data_size = enkf_config_node_get_data_size(config, 0);
-            const double mean =
-                0.0; /* Mean and std are hardcoded - the variability should be in the transformation. */
-            const double std = 1.0;
-
-            for (int i = 0; i < data_size; ++i) {
-                double random_number = enkf_util_rand_normal(mean, std, rng);
-                gen_kw_data_iset(gen_kw, i, random_number);
-                gen_kw_data_iset(gen_kw2, i, random_number);
-            }
-
-            rng_free(rng);
-        }
-        node_id_type node_id = {.report_step = 0, .iens = 0};
-
-        enkf_node_store(enkf_node, init_fs, node_id);
-        enkf_node_store(enkf_node2, init_fs, node_id);
-    }
-
-    {
-        enkf_main::ecl_write(res_config_get_ensemble_config(res_config),
-                             model_config_get_gen_kw_export_name(
-                                 res_config_get_model_config(res_config)),
-                             "simulations/run0", 0, init_fs);
-        test_assert_true(fs::exists("simulations/run0/parameters.txt"));
-    }
-    enkf_node_free(enkf_node);
-    enkf_node_free(enkf_node2);
-    enkf_fs_umount(init_fs);
-    verify_parameters_txt();
```

Comment:
This test was silly. However, we could perhaps use some test to verify the parameter file in a better way than here.